src/include/platform/CHIPDeviceConfig.h

```diff
@@ -178,7 +178,7 @@
  * persistent storage (e.g. by a factory provisioning process).
  */
 #ifndef CHIP_DEVICE_CONFIG_DEFAULT_DEVICE_PRODUCT_REVISION
-#define CHIP_DEVICE_CONFIG_DEFAULT_DEVICE_PRODUCT_REVISION 1
+#define CHIP_DEVICE_CONFIG_DEFAULT_DEVICE_PRODUCT_REVISION 0
```

Comment:
Should `DEFAULT_DEVICE_PRODUCT_REVISION` in `config/esp32/components/chip/Kconfig` be also set to 0 (default)?

Reply:
Done